frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -39,18 +38,27 @@ type MapProps = {
   onViewportChanged?: (e: MapViewportChangeEvent) => void;
   onMapClick?: (e: LeafletMouseEvent) => void;
   disableMapFilterBar?: boolean;
-  disabled?: boolean;
+  interactive?: boolean;
+  showParcelBoundaries?: boolean;
```

Comment:
- Renamed `disabled` to `interactive` - and flipped the associated logic.
- Added a toggle to show/hide the boundaries as the `Map` component is now used in various places; e.g. Landing Page and Submit Property form